composer/callbacks/checkpoint_saver.py

```diff
@@ -294,10 +294,15 @@ def __init__(
         num_concurrent_uploads: int = 1,
         upload_timeout_in_seconds: int = 3600,
     ):
+
         backend, _, local_folder = parse_uri(str(folder))
         if local_folder == '':
             local_folder = '.'
 
+        is_remote_folder = backend != ''
```

Comment:
nit: is there a more robust check we could have for this?

Reply:
https://github.com/mosaicml/composer/pull/3673/files#diff-b3be1fd7d6bb329e7dfcb406b7996e3ef170416faf5524caf50a451333b5f29eR348-R351 

We do it here, so I think it's fine